fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowAlterStmt.java

```diff
@@ -36,13 +37,13 @@
 /*
  * ShowAlterStmt: used to show process state of alter statement.
  * Syntax:
- *      SHOW ALTER TABLE [COLUMN | ROLLUP | MATERIALIZED_VIEW] [FROM dbName] [WHERE TableName="xxx"]
+ *      SHOW ALTER TABLE [COLUMN | ROLLUP | MATERIALIZED_VIEW | OPTIMIZE ] [FROM dbName] [WHERE TableName="xxx"]
```

Comment:
From the alter table statement, it seems that `OPTIMIZE` keyword occurs. Does it a good description?